docs/images/bluefin/framework.md

```diff
@@ -27,10 +29,25 @@ Open Source succeeds best whene contributors come together to solve a problem. T
 
 We want to allow Linux users to share this knowledge and expertise with each other to fulfill the true potential of the Framework. We'll [use science](https://www.youtube.com/watch?v=BABM3EUo990) and the wonders of modern automation to accomplish this!
 
-[Contributions welcome!](https://github.com/ublue-os/bluefin) 
+And lastly, right now `-framework` is a one-off done for Bluefin. It is entirely feasible for Universal Blue to adopt this across all the base images, but we'd need some help, devops ninjas, inquire within.
```

Comment:
instead of "one-off done" would it be clearer to say: "is a one-off created for Bluefin" ?